src/cursor/cur_bulk.c

```diff
@@ -208,13 +289,12 @@ __curbulk_insert_row_skip_check(WT_CURSOR *cursor)
 	 * until the bulk cursor is closed.
 	 */
 	CURSOR_API_CALL(cursor, session, insert, btree);
+	WT_STAT_FAST_DATA_INCR(session, cursor_insert_bulk);
 
-	WT_CURSOR_NEEDKEY(cursor);
-	WT_CURSOR_NEEDVALUE(cursor);
-
-	WT_ERR(__wt_bulk_insert_row(session, cbulk));
+	WT_CURSOR_CHECKKEY(cursor);
+	WT_CURSOR_CHECKVALUE(cursor);
```

Comment:
@agorrod, I've been migrating to this style over the past few months. I don't believe it matters whether we increment statistics before doing error checking (it's just as valid to count the total calls as to count the total "successful" calls), and it usually simplifies code paths at the end of the function.

In other words, get the simple stuff (that can't fail), out of the way as part of the function setup, and there will be fewer complications later on.

In this particular case, it makes less difference than in other cases I've seen, but I get to skip a `WT_ERR` call and I think the code is easier to read.